.travis.yml

```diff
@@ -65,3 +65,14 @@ script:
   - if [ -n "$CLANG_FORMAT_PATH" ]; then bin/clangfmt --test; fi
   - bin/scalafmt --test
   - bash sbt -J-Xmx2G rebuild "set scriptedBufferLog in sbtScalaNative := false" test-all
+
+before_cache:
+  - find $HOME/.sbt -name "*.lock" | xargs rm
+  - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
```

Comment:
Please exclude `.ivy2/cache/org.scala-native/` and `.ivy2/local/org.scala-native/` here. Those should always be rebuild from source and never cached.